### ISSUE TITLE
Create Accessibility Conformance Documentation

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-azurefunctions#2299]
GitHub Tags:#A11y_Azure Extensions for VS Code;#Azure Tools for VS Code;#A11yTCS;#A11yMAS;#MAC;#A11ySev1;

Accessibility Conformance Documentation requirement is waived for certain websites. To determine if your website qualifies for this waiver, read CELA Guidance​. Based on this guidance, if you think this requirement is waived for your website, then please add notes on why this website meets requirement for the waiver in this bug and resolve the bug. If after reading the guidance, you are unsure and have further questions, please contact CELA (Ben Rickert). If conformance documentation requirement is not waived based on guidance, then fix and close other bugs before starting accessibility conformance documentation. Any bug that will not be fixed and closed within SLA must be declared in the AAT. The SLA for this bug is also 60 days."

[Guide](https://microsoft.sharepoint.com/teams/ceacc/SitePages/Conformance%20Documentation.aspx) for creating conformance documentation

"When the AAT entry is completed, please resolve the bug with the AAT URL. Trusted tester will review the AAT and send comments if any. Once this review is completed, then mark the AAT entry for review in the AAT itself - this will send the entry to CELA for their review. When CELA publishes the document, the bug will be closed."

France Reporting Reqs: please refer to "French Decree Scope for Websites" page to determine if the product is in scope for French decree or not. If it is in scope, verify that the French Reporting Reqs section of the AAT is filled in (for questions on what and how to verify content in this section, please contact Chitra J.). If it is NOT in scope, then the checkbox in the AAT that reads "To the best of my knowledge, this website, service or application does not have French reporting obligations." must be checked.
